src/Libpng-2.0.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="Libpng-2.0" name="libpng License 2.0">
```

Comment:
The `license` tag should probably also include a `listVersionAdded="3.5"` value. (@jlovejoy has noted this on some of my PRs, so I'm noting it here) =)

Reply:
Added #795 to deprecate plain libpng identifier in favor of LibPNG-1.0